### ISSUE TITLE
Export Spritesheet reads all layers with the same name as the same layer

### ISSUE THREAD

**Issue by KashouC** [aseprite/aseprite#2200]
At least the preview will display the same image no matter which layer you choose if they are named the same thing. It seems to pick the bottom-most layer to display. I don't know if the export is bugged too or not.

![testt](https://user-images.githubusercontent.com/17066488/69428439-7a061900-0d31-11ea-9a8f-f27e432dd49d.gif)

**Comment by dacap** [comment]
Thanks for reporting this bug @KashouC, yeah, it's an old known issue, not too critical but I think it gets worse with layer groups (e.g. if a child uses the name than other child in other group).

Maybe something we could fix from the GUI, but it is a little more complex from the CLI. E.g. How to specify to export the specific "Layer 1" in your example from the CLI?

**Comment by KashouC** [comment]
Yeah I can see that issue with the CLI. In the actual program it would probably be somewhat simpler to fix by not explicitly using strings (I assume, I don't know the code). But the CLI would probably need some hackier solution unfortunately. 

I don't know how to solve it in a good way. A prompt saying there's X amount of layers and asking you to choose one or something? An alternative input method for layers to specify using numbers instead of strings? Like "1" looks for a layer named 1, while just typing 1 picks layer 1 from the bottom. Adding extensions to strings like "ExampleLayer"(1) or something to give the user a way to actually work around this case when it appears? Even if it might be an edge case.

That's pretty much what I can think of at least.

**Comment by Cleroth** [comment]
Perhaps Aseprite could just add the number itself automatically if you rename a layer to one that already exists.
